### ISSUE TITLE
Make view-files more friendly to non-interactive usage and machine readable output

### ISSUE THREAD

**Issue by adrianbroher** [nurdtechie98/drive-cli#91]
The current implementation forces some prompts on usage, even if the script is invoked within a pipeline where the prompt serves no purpose.

> Do you want to continue? [y/N]: y

The terminal is wiped, all useful information are cleared.

> Do you want to continue? [y/N]: n

The application reports "Aborted!" so did the application crash?  The exit code indicates so.

None of the `--name`, `--types` or `--pid` accepts the value from command line, but instead a prompt is forced.

Also the tabular output is unneccessary hard to parse to extract something like the ID for a subsequent invocation.

**Comment by vidstige** [comment]
Yes, the current interface is very difficult. I'm in favor of all of the above.
